backend/actions/Model/getDocuments.js

```diff
@@ -65,10 +65,13 @@ module.exports = ({ db }) => async function getDocuments(params) {
   const numDocuments = filter == null ?
     await Model.estimatedDocumentCount() :
     await Model.countDocuments(filter);
+
+  const indexes = await Model.listIndexes();
```

Comment:
Create a separate getIndexes() endpoint, and call that endpoint when the indexes modal opens. We're going to want to get more info on indexes later, most notably [index usage](https://www.mongodb.com/docs/manual/tutorial/measure-index-use/) and whether the index is in schema or not.